autoinstrumentation/nodejs/Dockerfile

```diff
@@ -21,3 +21,7 @@ FROM busybox
 COPY --from=build /operator-build/build/workspace /autoinstrumentation
 
 RUN chmod -R go+r /autoinstrumentation
+
+# Backward compatibility with old operator version
+RUN cp -r /autoinstrumentation/node_modules/@opentelemetry/auto-instrumentations-node/build/src/. /autoinstrumentation
+RUN ln /autoinstrumentation/register.js /autoinstrumentation/autoinstrumentation.js
```

Comment:
Shouldn't this be `/otel-auto-instrumentation-nodejs/autoinstrumentation.js`?

Reply:
![image](https://github.com/user-attachments/assets/cf697087-95e4-4b89-81b0-4a66dbbabd15)

Init container copies from autoinstrumentation to otel-auto-instrumentation-nodejs